src/components/common/SidebarCommon.vue

```diff
@@ -0,0 +1,332 @@
+<template>
+    <div class="flex-1 flex flex-col pt-10 overflow-y-auto">
+        <div class="flex items-center justify-center flex-shrink-0">
+            <img class="w-10 h-10" src="@/assets/img/plasm.png" />
+            <logotype />
+        </div>
+
+        <div class="p-4">
+            <button
+                type="button"
+                class="inline-flex justify-center w-full rounded-full border border-gray-300 dark:border-darkGray-600 px-4 py-3 bg-white dark:bg-darkGray-900 text-xs font-medium text-gray-700 dark:text-darkGray-100 hover:bg-gray-100 dark:hover:bg-darkGray-700 focus:outline-none focus:ring focus:ring-gray-100 dark:focus:ring-darkGray-600"
+            >
+                Plasm Network (Mainnet)
+                <!-- Heroicon name: solid/chevron-down -->
+                <svg
```

Comment:
Considering the SidebarComment component contains a lot of vector icon that is used by multiple components, I suggest moving the icons into a separate component for improved reusability.
Reference:
- https://medium.com/js-dojo/making-svg-icon-component-in-vue-cb7fac70e758
- https://vuejs.org/v2/cookbook/editable-svg-icons.html